src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp/Presentation/MainModel.cs

```diff
@@ -0,0 +1,69 @@
+//+:cnd:noEmit
```

Comment:
This will disable (if i'm not mistaken) the interpretaton of the `#if` statements by the templating engine.

Reply:
Unless the leading `+` is about that :)

Reply:
yeh the + turns it on :-)